config/locales/client.en.yml

```diff
@@ -584,6 +584,11 @@ en:
       date_filter: "Posted between"
       in_reply_to: "in reply to"
       filtered_flagged_by: "Flagged by"
+      unknown:
```

Comment:
This message reads a bit awkwardly with only one reviewable type in the list, it'd be nice to have a singular/plural version.